elephant/conversion.py

```diff
@@ -152,3 +155,704 @@ def binarize(spiketrain, sampling_rate=None, t_start=None, t_stop=None,
+
+    @property
+    def left_edges(self):
+        """
+        Returns an quantity array containing all left edges and
+        :attr:`num_bins` bins.
+
+        The left borders of all time steps between start and excluding stop
+        [start, stop) with number of bins from given input are regarded as
+        edges.
+        The border of the last bin is excluded.
+
+        Returns
+        -------
+        bin_edges : quantities.Quantity array
+            All edges in interval [start, stop) with :attr:`num_bins` bins
+            are returned as a quantity array.
+        """
+        return np.linspace(self.t_start, self.t_stop, self.num_bins,
+                           endpoint=False)
+
+    @property
+    def right_edges(self):
+        """
+        Returns the right edges with :attr:`num_bins` bins as a
+        quantities array.
+
+        The right borders of all time steps between excluding start and
+        including stop (start, stop] with :attr:`num_bins` from given input
+        are regarded as edges.
+        The border of the first bin is excluded, but the last border is
+        included.
+
+        Returns
+        -------
+        bin_edges : quantities.Quantity array
+            All edges in interval [start, stop) with :attr:`num_bins` bins
+            are returned as a quantity array.
+        """
+        return self.left_edges + self.binsize
+
+    @property
+    def center_edges(self):
+        """
+        Returns each center time point of all bins between start and stop
+        points.
+
+        The center of each bin of all time steps between start and stop
+        (start, stop).
+
+        Returns
+        -------
+        bin_edges : quantities.Quantity array
+            All center edges in interval (start, stop) are returned as
+            a quantity array.
+        """
+        return self.left_edges + self.binsize / 2
+
+    @property
+    def sparse_mat_unclip(self):
+        """
+        Getter for **unclipped** version of the sparse matrix.
+
+        Returns
+        -------
+        matrix: scipy.sparse.csr_matrix
+            Sparse matrix, counted, unclipped version.
+
+        See also
+        --------
+        scipy.sparse.csr_matrix
+        mat_unclipped
+        """
+        if self._sparse_mat_u is not None:
+            return self._sparse_mat_u
+
+    @property
+    def sparse_mat_clip(self):
+        """
+        Getter for **clipped** version of the sparse matrix.
+
+        Returns
+        -------
+        matrix: scipy.sparse.csr_matrix
+            Sparse matrix, binary, clipped version.
+
+        See also
+        --------
+        scipy.sparse.csr_matrix
+        mat_clipped
+        """
+        if self._sparse_mat_c is not None:
+            return self._sparse_mat_c
+        # Return sparse Matrix without storing
+        tmp_mat = self._sparse_mat_u.copy()
+        tmp_mat[tmp_mat.nonzero()] = 1
+        return tmp_mat
+
+    def store_sparse_mat_clip(self):
```

Comment:
You need to document in this method and the previous property how these relate.

Also, is there no method to remove it from memory?  Perhaps a method to add or remove it from memory, depending on an (optional) parameter?